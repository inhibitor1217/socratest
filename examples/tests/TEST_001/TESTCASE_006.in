1
double
2
int
