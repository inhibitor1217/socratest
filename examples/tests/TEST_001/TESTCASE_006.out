1.00
string
